lib/Sema/CodeSynthesis.cpp

```diff
@@ -826,6 +831,12 @@ static void synthesizeCoroutineAccessorBody(AccessorDecl *accessor,
                                             ASTContext &ctx) {
   assert(accessor->isCoroutine());
 
+  // FIXME: If we don't return here, then a throwing accessor can cause an
+  // infinite loop in SILGen
+  if (!accessor->getStorage()->hasStorage()) {
+    return;
+  }
```

Comment:
It's fine if something like this is necessary for incremental development, but I feel like the hack should go in SILGen, like maybe just not emitting a synthetic throwing accessor definition.

Reply:
@rjmccall Do you mean I should modify `SILGenFunction::emitCoroutineAccessor()` to not emit the couroutine accessor?

Reply:
Well, to emit it with a trivial body or something, yes.